packages/react-i18n/src/test/utilities.test.tsx

```diff
@@ -125,14 +125,23 @@ describe('translate()', () => {
       ]);
     });
 
-    it('uses the pluralization rules of the provided locale when a replacement named `count` is passed', () => {
-      const dictionary = {foo: {one: '{count} foo', other: '{count} foos'}};
-      expect(
-        translate('foo', {replacements: {count: 1}}, dictionary, locale),
-      ).toBe('1 foo');
-      expect(
-        translate('foo', {replacements: {count: 2}}, dictionary, locale),
-      ).toBe('2 foos');
+    describe('when a replacement named `count` is passed', () => {
+      it('uses the pluralization rules of the provided locale', () => {
+        const dictionary = {foo: {one: '{count} foo', other: '{count} foos'}};
+        expect(
+          translate('foo', {replacements: {count: 1}}, dictionary, locale),
+        ).toBe('1 foo');
+        expect(
+          translate('foo', {replacements: {count: 2}}, dictionary, locale),
+        ).toBe('2 foos');
+      });
+
+      it('formats the replacement `count`', () => {
+        const dictionary = {foo: {one: '{count} foo', other: '{count} foos'}};
+        expect(
```

Comment:
I might also test the `0` case. Just in case it passes `0.00` for some reason one day.

Reply:
Good point! I will add a test for that 👍